Sources/SwiftSyntax/Trivia.swift

```diff
@@ -10,6 +10,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+import Foundation
```

Comment:
SwiftSyntax is a low-level library that’s below `Foundation` in the stack and thus can’t depend on `Foundation` … unfortunately.